docs/README.md

```diff
@@ -0,0 +1,59 @@
+# How to document a Giant Swarm product
+
+We provide product-specific documentation inside the repository of the product, using a top-level directory named `docs`.
+
+The content within this directory is intended primarily for reading on the `giantswarm.io` website. To enable useful rendering in that context, please adhere to the following guidelines.
+
+## Markdown content
+
+Documentation content is in [Markdown](https://daringfireball.net/projects/markdown/syntax) format. Github-flavored Markdown extensions such as fenced code blocks and tables might be applied.
+
+## Index files
+
+We make use of the fact that when entering a directory of a repository, GitHub automatically renders the `README.md` file content if available.
+
+If no `README.md` file is available in the `docs` folder or any subfolder, a table of contents page is automatically generated for browsing that folder within the `giantswarm.io` website. To provide a custom table of contents, provide a `README.md` file and set links to all sub-pages.
```

Comment:
best recommend to always have a README.md so we have a better/coherent UX on GitHub